contacts/style.css

```diff
@@ -0,0 +1,205 @@
+:root {
+    --main-background-color: #ffdd99;
+    --other-background-color: #336600;
+    --text-color: #ffdd99;
+    --nice-black-color: #001a13;
+    --pop-color: #15945f;
+}
+
+.align-center {
+    text-align: center;
+}
+
+html {
+    background-color: var(--main-background-color);
+}
+
+header {
+    background-color: var(--other-background-color);
+    border: 2px solid black;
+    border-radius: 25px;
+    margin: 2px;
+}
+
+header h1 {
+    padding: 0px;
+    font-size: 3.5em;
+    color: var(--main-background-color)
+}
+
+nav {
+    margin: 2px;
+}
+
+nav ul {
+    background-color: var(--other-background-color);
+    margin: 0px;
+    padding: 10px;
+    overflow: hidden;
+    border: 2px solid black;
+    border-radius: 25px;
+}
+
+nav ul li {
+    display: inline;
+    font-size: 2em;
+    padding: 40px;
+    color: var(--text-color);
+}
+
+nav ul li:hover {
+    background-color: var(--nice-black-color);
+}
+
+nav ul li a {
+    color: var(--text-color);
+    text-decoration: none;
+}
+
+.separator {
+    display: inline;
+    font-size: 2em;
+    padding: 0px;
+    color: var(--text-color);
+}
+
+footer {
+    background-color: var(--other-background-color);
+    padding: 10px;
+    border: 2px solid black;
+    border-radius: 25px;
+    margin: 2px;
+}
+
+footer p {
+    padding: 50px;
+    margin: 20px;
+    text-align: right;
+    display: inline;
+    color: var(--text-color);
+}
+
+.person-field {
+    padding: 10px;
+    border: 2px solid black;
+    border-radius: 25px;
+    margin: 2px;
+    display:flex;
+    align-items: center;
+    padding: 20px;
```

Comment:
Padding is set 2 times on this class. The last padding value will overwrite all values that were set previously. Remove unwanted values